### DISCUSSION TITLE
Are breadcrumbs getting automatically limited when exceeding a certain limit?

### DISCUSSION THREAD

**Discussion by Kjir** [getsentry/support#5] [Support] [answered]
Hey all,
I am adding some breadcrumbs around our application to be able to better trace what happened before an error. However since this is a single page application, the number of breadcrumbs are potentially infinite depending on how many things are done with the app before it encounters an error.

So I am wondering: is the SDK limiting automatically the number of breadcrumbs to stay within a reasonable size? Or should I implement some other strategy?

**Comment by HazAT** [comment, marked as answer]
Yes, our SDKs keep a circular buffer of breadcrumbs; the exact number of how many we keep depends on the SDK.
It's usually between 100 and 200 breadcrumbs. You have options to increase this number but check out our docs on how to do it.

**Reply by Kjir** [reply]
That's perfect, thanks!
